system-int-test/test/cmr/system_int_test/search/autocomplete/suggestion_reindex_test.clj

```diff
@@ -61,6 +66,27 @@
 (def sk6 (umm-spec-common/science-keyword {:Category "EARTH SCIENCE"
                                            :Topic "Popular"
                                            :Term "Omega"}))
+
+(def sk7 (umm-spec-common/science-keyword {:Category "missing"
```

Comment:
I'm guessing nil values aren't possible here or do not need to be checked? only these anit-value string values?

Reply:
Not sure if nil is possible here, or in general. I'll try it and see what happens and adjust if necessary.

Reply:
nil cannot be used here. It throws an exception when ingesting so I think it's safe to ignore that case.